lib/src/db/annotations.dart

```diff
@@ -49,14 +49,16 @@ class Kind {
 /// The type used for id's of an entity.
 class IdType {
   /// Use integer ids for identifying entities.
-  static const IdType Integer = const IdType(1);
+  static const IdType Integer = const IdType('Integer');
 
   /// Use string ids for identifying entities.
-  static const IdType String = const IdType(2);
+  static const IdType String = const IdType('String');
 
-  final int _type;
+  final core.String _type;
```

Comment:
If you want to make a cl which runs dartfmt, then that's the only thing you should do. Not just out of a 1000+ line diff, suddenly change small subtle things like this!

Reply:
There is a `commits` tab which lets you look at each commit. Is nice...

Reply:
Is there a problem w/ this code?

Reply:
This CL diff doesn't show the use sites of `IdType`, so I don't know. Probably not a problem. Though changing types in classes exposed to users is different to running dartfmt on the sources, so I would like to actually see what has changed semantically.

Reply:
Ack